### PR TITLE
[SYCLomatic] DPCT/C2S bash auto-complete

### PR DESCRIPTION
**Pull request by zhiweij1**

### CODE REVIEW

File: clang/tools/dpct/bash-autocomplete.sh [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# Please add "source /path/to/bash-autocomplete.sh" to your .bashrc to use this.
```
Comment (zhimingwang36) [inline clang/tools/dpct/bash-autocomplete.sh:1]: Please consider to install this script to install folder when run "ninja install-dpct"

File: llvm/include/llvm/DPCT/DPCTOptions.inc [hunk 2]
```diff
@@ -0,0 +1,399 @@
+#ifndef DPCT_OPT_TYPE
```
Comment (zhimingwang36) [inline llvm/include/llvm/DPCT/DPCTOptions.inc]: Please consider to put the DPCTOptoins.inc file into clang/include/clang/DPCT?

**Review by zhimingwang36 — APPROVED** [review]
LGTM

**Review by tomflinda — APPROVED** [review]
LGTM

**Pull request merged**